App/php/add-event.php

```diff
@@ -0,0 +1,17 @@
+<?php
+require_once "config.php";
+session_start();
+
+$title = isset($_POST['title']) ? $_POST['title'] : "";
+$start = isset($_POST['start']) ? $_POST['start'] : "";
+$end = isset($_POST['end']) ? $_POST['end'] : "";
+$user_email = $_SESSION['email'];
+
+$sqlInsert = "INSERT INTO Calendar (title,start,end,u_email) VALUES ('".$title."','".$start."','".$end ."','".$user_email."')";

```

Comment:
Not sure if this works correctly cause of the weird quotation marks. Change it to
```php
$sqlInsert = "INSERT INTO Calendar (title, start, end, u_email) VALUES ('$title', '$start', '$end', '$user_email')";
```